aten/src/ATen/native/cuda/GridSampler.cu

```diff
@@ -303,6 +303,248 @@ namespace {
 
 // Note [Passing pointer and offset to fastAtomicAdd]
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
You broke up the comment `Note [Passing pointer and offset to fastAtomicAdd]`.  Please restore this header to above the comment body.